rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/frontend/ResourceManager.java

```diff
@@ -83,9 +100,21 @@
      * @param nodesRecoverable whether the nodes can be recovered in case of a scheduler crash
      * @return true if a new node source was created successfully, runtime exception otherwise
      */
+    @Deprecated
     BooleanWrapper createNodeSource(String nodeSourceName, String infrastructureType, Object[] infrastructureParameters,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)